src/zenml/stack/stack.py

```diff
@@ -358,10 +358,25 @@ def deploy_pipeline(
         )
         start_time = time.time()
 
+        original_cache_boolean = pipeline.enable_cache
+        if "enable_cache" in runtime_configuration:
+            logger.info(
+                "Runtime configuration overwriting the pipeline cache settings"
+                " to enable_cache=`%s` for this pipeline run. The default "
+                "caching strategy is retained for future pipeline runs.",
+                runtime_configuration["enable_cache"],
+            )
+            pipeline.enable_cache = runtime_configuration.pop("enable_cache")
```

Comment:
Is there a reason why the `enable_cache` is popped from the `runtime_configuration`? If it is retained, it could be cool to show what users opted for in terms of caching within the MLMD (through the context of runtime configurations in the nodes).

Reply:
I was worried of how this might affect things downstream, I just tried it without popping and seems to work well.